recipes/application_python.rb

```diff
@@ -95,6 +95,7 @@
 if node.deep_fetch(stackname, 'code-deployment', 'enabled')
   node[stackname][node[stackname]['webserver']]['sites'].each do |port, sites|
     sites.each do |site_name, site_opts|
+      next if site_opts['repository'] == ''
```

Comment:
if repository is unset that check would fail, that's why we default to an empty string